src/integration/pages/articlePage/__snapshots__/mundo.test.js.snap

```diff
@@ -1985,7 +1985,7 @@ exports[`Given I am on a Mundo Article page When I view the source code in the b
             crossorigin="anonymous"
             defer
     >
-      [21]
+      [19]
```

Comment:
I'm not sure why these numbers are being updated in these snapshots for the integration tests. Is it something to do with webpack chunking differently between test runs?